app/Views/Templates/components/forms/chip/quick-add.blade.php

```diff
@@ -0,0 +1,52 @@
+@props([
+    'type' => '', //status, milestone, effort, priority, user, sprint
+    'parentId' => '',
+    'postUrl' => '',
+])
+
+@php
+    $inputName = $type == 'milestone' ? 'headline' : 'name';
+@endphp
+
+<div {{ $attributes->merge([ 'class' => 'mx-2' ]) }}>
```

Comment:
From above: You can take the attributes array and look for keys() that start with 
'hx'. Then take all those keys and add them below to the form tag. 

I believe $attribute is a collection so you can do:

```php 

$hxAttributes = $attributes->filter(function($field, $key){            
    if(Str::startsWith($key, 'hx')) {
         return;
    }
});
 ```

Reply:
PS: You would want to only show the non hx tags up here in the div tag